assets/scss/_registry.scss

```diff
@@ -34,6 +34,18 @@
   $default-otel-badge-bg: $yellow;
   color: color-contrast($default-otel-badge-bg);
   background-color: $default-otel-badge-bg;
+
+  background-color: var(--bs-body-bg) !important;
+
+  [data-bs-theme="light"] & {
+    background-color: #fff !important;
+    color: inherit;
+  }
+
+  [data-bs-theme="dark"] & {
+    background-color: #212529 !important;
+    color: inherit;
+  }
```

Comment:
can you make sure that this is only applied to the labels on the registry entries, such that if we plan to use labels somewhere else as well we do not run into isssues.